packages/aila/src/core/prompt/builders/AilaLessonPromptBuilder.ts

```diff
@@ -89,6 +89,10 @@ export class AilaLessonPromptBuilder extends AilaPromptBuilder {
         : undefined,
       lessonPlanJsonSchema: JSON.stringify(LessonPlanJsonSchema),
       llmResponseJsonSchema: JSON.stringify(LLMResponseJsonSchema),
+      isUsingStructuredOutput:
```

Comment:
could go for the more imperative `useStructuredOutput` for brevity ... or `shouldUse...`

Reply:
I think we might want to create a prompt variant for this eventually seeing as the prompt does change depending on this variable